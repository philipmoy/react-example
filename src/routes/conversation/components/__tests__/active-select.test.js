import React from 'react';
import { shallow } from 'enzyme';

import ActiveSelect from '../active-select';

describe('ConversationActiveSelect', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <ActiveSelect />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
