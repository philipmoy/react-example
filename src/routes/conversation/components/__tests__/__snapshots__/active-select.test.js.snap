// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationActiveSelect renders correctly 1`] = `
<CustomSelect
  onSelect={[Function]}
  options={
    Array [
      Object {
        "value": "low",
      },
      Object {
        "value": "medium",
      },
      Object {
        "value": "high",
      },
    ]
  }
  renderOption={[Function]}
/>
`;
